include/charStringLiteralDfa.h

```diff
@@ -0,0 +1,16 @@
+#include "dfa.h"
+#include <string>
+
+#define octalLimit 3
+
+class CharStringLiteralDfa : public Dfa {
```

Comment:
I'm starting to make al these final classes :3 so if you could make the change for the new ones thanks :D 

class CharStringLiteralDfa final : public Dfa {